src/Compilers/VisualBasic/Portable/Symbols/Tuples/TuplePropertySymbol.vb

```diff
@@ -132,5 +132,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         Public Overrides Function GetAttributes() As ImmutableArray(Of VisualBasicAttributeData)
             Return Me._underlyingProperty.GetAttributes()
         End Function
+
+        Public Overrides ReadOnly Property IsRequired As Boolean
+            Get
+                Return _underlyingProperty.IsRequired
```

Comment:
> _underlyingProperty.IsRequired [](http://example.com/codeflow?start=22&length=31)

Are we testing effect of this fix?

Reply:
I'll add a direct test, I don't expect this to be directly observable.

Reply:
> I don't expect this to be directly observable except via the public API.

I think it can be observed by explicitly using an object creation expression rather than a tuple literal. At least it is observable in C# this way.